crates/sui-replay/src/replay.rs

```diff
@@ -779,8 +779,21 @@ impl LocalExec {
 
         trace!(target: "replay_gas_info", "{}", Pretty(&gas_status));
 
-        if let ProgrammableTransaction(pt) = transaction_kind {
-            trace!(target: "replay_ptb_info", "{}", Pretty(&pt));
+        if let ProgrammableTransaction(ref pt) = transaction_kind {
+            trace!(target: "replay_ptb_info", "{}", Pretty(&(pt.clone(), executor.dev_inspect_transaction(&self, protocol_config,
+                metrics,
+                expensive_checks,
+                &certificate_deny_set,
+                &tx_info.executed_epoch,
+                epoch_start_timestamp,
+                CheckedInputObjects::new_for_replay(input_objects),
+                tx_info.gas.clone(),
+                SuiGasStatus::new(tx_info.gas_budget, tx_info.gas_price, rgp, protocol_config)?,
+                transaction_kind.clone(),
+                tx_info.sender,
+                *tx_digest,
+                true
```

Comment:
nit, somehow label this value (bound to a local or a comment)

Reply:
updated